tests/conftest.py

```diff
@@ -0,0 +1,16 @@
+import os
+
+import pytest
+
+
+@pytest.fixture
+def env_setup():
+    old_pwd = os.environ.pop("PASSWORD", None)
+    from conda_forge_tick.env_management import sensitive_env
+
+    with sensitive_env() as env:
+        env["PASSWORD"] = "unpassword"
+        env["PASSWORD2"] = "unpassword"
+    yield
```

Comment:
Does this yield need to go in one leve?

Reply:
ahhh nvm

Reply:
This is not going to work. Removed my commits for now